src/js/views/patients/actions/actions_views.js

```diff
@@ -254,6 +254,82 @@ const DueComponent = Component.extend({
   },
 });
 
+// Every 15 mins for 24 hours starting at 7am
+const start = moment({ hour: 6, minute: 45, second: 0 });
+
+const times = _.times(96, function() {
+  return { time: start.add(15, 'minutes').format('LT') };
+});
+
+times.unshift({ time: null });
+
+const DueTimeComponent = Droplist.extend({
+  isCompact: false,
+  isSelectlist: true,
+  getClassName(time) {
+    const isCompact = this.getOption('isCompact');
+
+    if (time === null && isCompact) {
+      return 'button-secondary--compact actions__time is-icon-only';
+    }
+    if (isCompact) {
+      return 'button-secondary--compact actions__time ';
+    }
+
+    return 'button-secondary actions__time w-100';
+  },
+  getTemplate(time, isCompact) {
+    if (!time && this.getOption('isCompact')) {
+      return hbs`{{far "clock"}}`;
+    }
+
+    if (!time) {
+      return hbs`{{far "clock"}}{{@intl.patients.actions.actionsViews.dueTimeComponent.placeholderText}}`;
+    }
+
+    return hbs`{{far "clock"}} {{time}}`;
+  },
+  viewOptions() {
+    const time = (this.getState('selected')) ? this.getState('selected').get('time') : null; // not great bob
+
+    return {
+      className: this.getClassName(time),
+      template: this.getTemplate(time),
+    };
+  },
+  picklistOptions: {
+    headingText: i18n.dueTimeComponent.headingText,
+    placeholderText: i18n.dueTimeComponent.placeholderText,
+    isSelectlist: true,
+    getItemFormat(item) {
+      const time = item.get('time');
+
+      if (!time) {
+        return i18n.dueTimeComponent.clear;
+      }
+
+      return time;
```

Comment:
actually second look I think this is the only place that needs `.format('LT')` everything else can be `HH:mm:ss` which should remove the need for reformatting the time in a few places.

Reply:
so if I don't have `format` up on [line 261](https://github.com/RoundingWell/care-ops-frontend/pull/210/files#diff-0a759ec2641c5a9b600cafd635ba96faR261) it generates 95 of the same time (6:45am), which is why I've ended up needing to format everywhere

Reply:
Right so `.add` overwrites the moment, but there it should probably be `.format('HH:mm:ss')` so that everywhere the data is the same and the only place we're using `LT` is directly before displaying the data.